docs/api/api.yml

```
@@ -259,6 +285,33 @@ paths:
         409:
           description: The bag containing the item is `INACTIVE`.
 
+  /stores/{store}/bags/filesizes/{uuid}/{path}:
```

@janvanmansum @vesaakerman applying the same suggestions as above, I propose to change this to

```suggestion
  /stores/{store}/bags/{uuid}/filesize/{path}:
```